src/genie/libs/parser/aos/show_vlan.py

```diff
@@ -0,0 +1,108 @@
+"""show_vlan.py
+
+"""
+import re
+import logging
+
+from genie.metaparser import MetaParser
+from genie.libs.parser.utils.common import Common
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+logger = logging.getLogger(__name__)
+
+
+def regexp(expression):
+    def match(value):
+        if re.match(expression,value):
+            return value
+        else:
+            raise TypeError("Value '%s' doesnt match regex '%s'"
+                              %(value, expression))
+    return match
+
+# ====================================================
+#  schema for show vlan
+# ====================================================
+class ShowVlanSchema(MetaParser):
+    """Schema for show vlan"""
+    schema = {
+        'vlans':{
+            Any():{
+                Optional('vlan_id'): str,
+                Optional('name'): str,
+                Optional('status'): str,
+                Optional('voice'): str,
+                Optional('jumbo'): str,
+               
+                    }
+                },
+            }
+
+# ====================================================
+#  parser for show vlan
+# ====================================================
+class ShowVlan(ShowVlanSchema):
+    """Parser for show vlan"""
+    cli_command = 'show vlan'
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+#   This matches the report portion of the show vlan command.
```

Comment:
I think that the file could be cleaned up a bit by having the comments at the appropriate level of indentation for the code